tests/ecosystem/deployment/test_ocs_basic_install.py

```diff
@@ -177,6 +202,30 @@ def test_deployment(self):
         log.info(f"Waiting {wait_time} seconds...")
         time.sleep(wait_time)
 
+        # Create MDS pods for CephFileSystem
+
+        self.fs_data = defaults.CEPHFILESYSTEM_DICT.copy()
+        self.fs_data['metadata']['namespace'] = ENV_DATA['cluster_namespace']
+
+        global CEPH_OBJ
+        CEPH_OBJ = OCS(**self.fs_data)
+        CEPH_OBJ.create()
+        assert POD.wait_for_resource(
+            condition='Running', selector='app=rook-ceph-mds',
+            resource_count=2, timeout=600
+        )
+
+        # Check for CephFilesystem creation in ocp
+        cfs_data = CFS.get()
+        cfs_name = cfs_data['items'][0]['metadata']['name']
+
+        # Check successful creation of cephfilesystem from ceph backend
+        ct_pod = pod.get_ceph_tools_pod()
+        cmd_out = ct_pod.exec_ceph_cmd(ceph_cmd="ceph fs ls ")
+
+        if cfs_name in [pool.get('name') for pool in cmd_out]:
```

Comment:
Hi @nehaberry you can also use validate_cephfilesystem(cfs_name) from helpers.py

Reply:
oh. I didnt check that function.

Reply:
@prsurve The functions were added yesterday, hence sorry I missed them. does this look ok ?

# Create MDS pods for CephFileSystem

        self.fs_data = defaults.CEPHFILESYSTEM_DICT.copy()
        self.fs_data['metadata']['namespace'] = ENV_DATA['cluster_namespace']

        global CEPH_OBJ
        CEPH_OBJ = OCS(**self.fs_data)
        CEPH_OBJ.create()
        assert POD.wait_for_resource(
            condition='Running', selector='app=rook-ceph-mds',
            resource_count=2, timeout=600
        )

        # Check for CephFilesystem creation in ocp and Ceph side
        cfs_data = CFS.get()
        cfs_name = cfs_data['items'][0]['metadata']['name']
        helpers.validate_cephfilesystem(cfs_name)

I tested and it passed

Reply:
@prsurve changed and resent the PR. Please review and let me know.

Note: I didn't use helpers.create_cephfilesystem() to create the MDS pods as i asserts upon failure. Currently, I dont want to fail the wholedeployment in case MDS pod creation faces any issue. I am just logging the error message.

Reply:
Run result:

18:16:47 - MainThread - tests.helpers - INFO - ocsci-cephfs
18:16:47 - MainThread - tests.helpers - INFO - FileSystem got created from Ceph Side
18:16:47 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/nberry/aws-install/jun13-4/auth/kubeconfig get CephFileSystem ocsci-cephfs -o yaml
18:16:49 - MainThread - tests.helpers - INFO - Filesystem got created from kubernetes Side
18:16:49 - MainThread - tests.ecosystem.deployment.test_ocs_basic_install - INFO - MDS deployment is successful
18:16:49 - MainThread - tests.ecosystem.deployment.test_ocs_basic_install - INFO - Done creating rook resources, waiting for HEALTH_OK
18:16:49 - MainThread - utility.utils - INFO - Executing command: oc wait --for condition=ready pod -l app=rook-ceph-tools -n openshift-storage --timeout=120s
18:16:50 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage get pod -l 'app=rook-ceph-tools' -o jsonpath='{.items[0].metadata.name}'
18:16:51 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage exec rook-ceph-tools-76c7d559b6-2mc2n ceph health
18:16:55 - MainThread - tests.ecosystem.deployment.test_ocs_basic_install - INFO - HEALTH_OK, install successful.
PASSED                                                                                                        [100%]

==================================== 1 passed, 15 deselected in 2282.31 seconds =====================================
(ocs-ci) [nberry@localhost ocs-ci]$